utilities/deep_inspection.py

```
@@ -96,14 +107,120 @@ def create_di_connection_secret(
+                )
+
+    except TimeoutExpiredError:
+        raise ConversionError(
+            conversion_name=conversion.name,
+            phase=last_phase or "Unknown",
+            message=f"Did not reach phase '{phase}' within {timeout}s",
+        )
+
+
+def wait_for_di_snapshot(
+    source_provider: "VMWareProvider",
```

Ruff (0.15.20)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)